test/test_publish.py

```diff
@@ -15,21 +10,8 @@
 TEST_SECRET_KEY = 'fake-secret-key'
 
 
-@pytest.fixture
```

Comment:
Were we never actually testing against this fixture?

Reply:
Correct. For this test, the actual `publish_to_s3` method itself is mocked, so it never even calls AWS and there is no need to mock the bucket. There are other tests in `test_s3publisher.py` that actually make the calls and rely on a mocked bucket.

Reply:
👍